spec/lib/oembed_resolver_spec.rb

```diff
@@ -42,6 +42,15 @@
       end
     end
 
+    context 'when supplied a Google Slide URL' do
+      let(:pasted_link) { 'https://docs.google.com/presentation/d/e/2PACX-1vSEEzfWytpa8lMbyCYSHDec3_mLuJAhXWt_gaST6JOoNN0zGfBHToiA45ZoCUIR7cwCzRsn3Hj6YNl7/pub?start=false&loop=false&delayms=3000' }
+      let(:expected_embed_code) { "<iframe src=\"#{pasted_link.gsub('pub', 'embed')}\" frameborder=\"0\" width=\"960\" height=\"569\" allowfullscreen=\"true\" mozallowfullscreen=\"true\" webkitallowfullscreen=\"true\"></iframe>" }
```

Comment:
Let's no repeat tested code in test itself - just paste here correct value as constant string